src/AaveV3EthFraxListing_20230619/ADD_FRAX_ETHEREUM_V3.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Add FRAX Ethereum Aave v3 
+discussions: https://governance.aave.com/t/arfc-add-frax-to-aave-v3-ethereum/13051
+shortDescription: Add FRAX Ethereum Aave v3 
+author: Flipside & TokenLogic
+---
+
+# Simple Summary
+
+This publication presents the community an opportunity to add FRAX to the Ethereum v3 Liquidity Pool.
+
+# Motivation
+
+FRAX can use its Lending AMO (similar to Maker’s DAI Direct Deposit Module) to mint protocol controlled FRAX to be lent out on Aave Protocol. Frax Finance has in the past done so in Aave v2 on Ethereum and has already stated on the Aave governance forum an interest in doing so on other deployments once FRAX is added.
+
+The Frax Finance team could deploy a similar Aave Lending AMOs to Aave v3 after FRAX is listed.This will provide Aave users with access to FRAX and present an alternative to the four USD stable coins on Aave v3.
```

Comment:
```suggestion
The Frax Finance team could deploy a similar Aave Lending AMOs to Aave v3 after FRAX is listed. This will provide Aave users with access to FRAX and present an alternative to the four USD stable coins on Aave v3.
```